packages/utils/src/api/indexer/context.tsx

```diff
@@ -2,25 +2,36 @@ import { createContext, ReactNode, useCallback, useState } from "react";
 
 type IndexerAPIContextType = {
   indexerUrl: string;
-  headers?: RequestInit["headers"],
-  setIndexerUrl: (url: string) => void
+  headers?: RequestInit["headers"];
+  credentials?: RequestInit["credentials"];
+  setIndexerUrl: (url: string) => void;
 };
 
 const initialState: IndexerAPIContextType = {
   indexerUrl: "",
-  setIndexerUrl: () => { }
+  credentials: "omit",
```

Comment:
Let's remove this line and instead, provide it to the provider as prop.

https://github.com/cartridge-gg/controller/blob/011bede8b9959e78a4e68633e502f67a26645388/packages/profile/src/components/context/provider.tsx#L18
```ts
<IndexerAPIProvider credentials="omit" />
```